docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+- Using a mobile app to transfer money
+- Purchasing concert tickets
+- Booking a vacation
+- Ordering a meal delivery
+- Filing an expense report
+- Creating a new employee onboarding process
+
+In Temporal, a Workflow defines the steps your application takes to complete a task, much like a blueprint for your application. 
+It’s your business logic, written in code, outlining each step in your process. 
+This could be moving money between bank accounts, processing orders, deploying cloud infrastructure, or training an AI model. 
+
+Temporal isn’t a no-code workflow engine. 
+It is **Workflows-as-Code**. 
+Instead of dragging and dropping steps in a visual interface, you write your Workflows in code in your favorite programming language, code editor, and other tools. 
+This gives you full control and flexibility over your business processes, allowing you to build exactly what you need. 
+
+Workflows are resilient and stateful. 
+With Temporal, they are built to handle problems smoothly. 
+They can pause, pick up where they left off, or try again automatically. 
+If a failure or a crash happens, this ensures that your business processes keep running without interruptions.
+
+### Activities
+Activities are the individual tasks within your Workflow.
+
+They handle jobs that interact with the outside world, like sending an email, making a network request, writing to a database, or calling an API. 
+These tasks are where issues might occur, such as network failures or service errors. 
+Activities are pieces of code, either functions or methods depending on the programming language, that perform these tasks. 
+Activities are invoked in Workflow code. 
+
+If an Activity fails, Temporal automatically retries it based on your configuration. 
+You have full control over how often and how many times these retries should happen. 
+You can also define specific timeouts for each Activity, triggering a retry if the task takes too long to complete.
+
+### SDKs
+Developers create Temporal applications by writing code, just like you would to create any other software. 
+To do this, you use an Temporal SDK, which stands for software development kit. 
+
+The Temporal SDK is an open source library that developers install in their application to use Temporal. 
+It provides everything needed to build Workflows, Activities, and various other features in a specific programming language.
+
+Temporal currently offers six SDKs: Java, Go, TypeScript, .NET, Python, and PHP. 
+Installing the SDK allows you to easily add Temporal to your current projects without having to change the tools you're already using. 
+Temporal fits right into your existing tech stack.
+
+The Temporal SDK also makes it easier to implement complex tasks such as handling jobs either in order or at the same time (sync/async execution), automatically retrying failed tasks, scheduling tasks to run regularly, managing complex workflows (SAGA compensation), and managing errors (advanced error handling).
```

Comment:
I don't think you need 'managing errors (advanced error handling)'. I think just 'managing errors' is fine.

Reply:
Yeah I believe I took this from a list somewhere and turned it into a sentence that's more easy to take in for this context. Agreed that it's basically the same thing.